cli.md

```diff
@@ -80,6 +80,29 @@ The `executable` could involve multiple command line tokens (e.g., `npm run` or
 }
 ```
 
+## Configuring OpenAI
+
+You can optionally allow codemods to access OpenAI by adding the following environment variable:
+```bash
+CODEMODDER_OPENAI_API_KEY=<KEY>
+```
+
+In the case of problems communicating with an AI subprocessor, all other codemods should still run normally.
+
+### Using Azure OpenAI
+
+Codemods request access to OpenAI models by name, and by default are given access to types that talk to OpenAI directly. It's possible to override the default behavior to use Azure OpenAI in codemods by specifying the following environment variables:
```

Comment:
I think we should say that codemodder will give precedence to Azure OpenAI if both API keys are present.